### PR TITLE
If the system has 7zip, use the system's 7zip.

### PR DESCRIPTION
**Pull request by Hill-98**
This is a PR like #23, although the #28 issue is weird, I think it's been fixed by #32, so it's time to reconsider.

Compared to #23, this PR can detect any 7zip installed on the system, not just `Program Files`.

cc: @iamCristYe 

edit: I referenced the wrong issue

### CODE REVIEW

**Comment by iamCristYe** [comment]
@Hill-98 this is much better. Thanks

File: mpv-root/installer/updater.ps1 [hunk 1]
```diff
@@ -1,15 +1,31 @@
+$fallback7z = Join-Path (Get-Location) "\7z\7zr.exe";
+
+function Get-7z {
+    $7z_command = Get-Command -CommandType Application -ErrorAction Ignore 7z.exe
+    if ($7z_command) {
+        return $7z_command.Source
+    }
+    $7zdir = Get-ItemPropertyValue -ErrorAction Ignore "HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\7-Zip" "InstallLocation"
+    if ($7zdir -and (Test-Path $7zdir) -and (Test-Path (Join-Path $7zdir "7z.exe"))) {
```
Comment (shinchiro) [inline mpv-root/installer/updater.ps1:9]: Why there's 2 checks on `Test-Path`? Single check with `(Test-Path (Join-Path $7zdir "7z.exe")` should be sufficient

Comment (Hill-98) [reply mpv-root/installer/updater.ps1:9]: Oh yeah, this is redundant.

**Comment by shinchiro** [comment]
LGTM, I didnt test the checking on installed 7zip since I dont have it installed. 

Lastly, change the commit message to `updater: use system 7zip installed if available`

**Pull request merged**